Jenkinsfile

```diff
@@ -0,0 +1,238 @@
+def TIKV_BRANCH = "master"
```

Comment:
Why we need to put this jenkinsfile in lightning repo? Currently only `binlog` use similar method to maintain ci pipeline scripts are we going to put ci scripts to other repo too? And we have a repo https://github.com/pingcap/sre that used to maintain all sre scripts(include ci scripts).
In my experience in maintaining CI, I think it's not a good idea to put this script here. In most time, these scripts are managed by QA or EE teams, only few changes are related to test logic changes, and the CI maintainers should take responsibility to manage this instead of developers

Reply:
yes we could put it in SRE. 

wdyt @3pointer

Reply:
pingcap/SRE#339. 

Will delete if that PR is merged.